df.units.xml

```diff
@@ -2260,7 +2260,7 @@
             <flag-bit/>
             <flag-bit name='lightly_tap'/>
             <flag-bit/>
-            <flag-bit/>
+            <flag-bit name='is_sparring'/>
```

Comment:
the bay12 name for this flag bit is `SPARRING_HIT`

the rest of these are, in order:
* `FORCE_SMALL_AIM_MINUS`
* `CAN_GET_MOMENTUM_BONUS`
* `NO_PARRY`
* `NO_BLOCK`
* `DOES_NOT_STOP_WAIT`
* `QUICK`
* `HEAVY`
* `WILD`
* `PRECISE`
* `INDEPENDENT_MULTIATTACK`
* `BAD_MULTIATTACK`
* `ZERO_CRITICALS`
* `NO_LETHAL_INTENT`
* `AUTOMATIC_HIT`
* `SPARRING_HIT`

Reply:
also, the bay12 name for `unit_action_data_attack` is `unit_move_attackst`

Reply:
What's the last one (`spar_report`?) Should `charge` be changed to `independent_multiattack` given how different it is?

If we're going to change `unit_action_data_attack` to `unit_move_attackst`, we should probably change all of them.

Reply:
> If we're going to change `unit_action_data_attack` to `unit_move_attackst`, we should probably change all of them.

i don't disagree but i won't have time to address this until after 50.12 has been released and i'm done with the usual post-release processes, which typically takes one to two weeks